src/console/Handlers.cpp

```diff
@@ -15,8 +16,28 @@ int32_t OnIdle(const EVENT_DATA_IDLE* data, void* param) {
 }
 
 int32_t OnKeyDown(const EVENT_DATA_KEY* data, void* param) {
-    // TODO
-    return 1;
+    bool active;
+    
+    if (data->key == ConsoleGetHotKey() && ConsoleAccessGetEnabled()) {
+        active = ConsoleGetActive() == false;
+
+        ConsoleSetActive(active);
+
+        if (!active) {
+            // TODO
+            // ResetHighlight();
+        }
+
+        return 0;
+    }
+    else if (EventIsKeyDown(ConsoleGetHotKey()) || !ConsoleGetActive()) {
```

Comment:
```suggestion
    }

    if (EventIsKeyDown(ConsoleGetHotKey()) || !ConsoleGetActive()) {
```

Reply:
We can treat these early conditionals as guard conditions. In a guard condition, when the condition is met and the condition returns from the function, there's no need to use `else` to chain conditions. Each condition can just be fully separate `if` block, as I've suggested in my review.

Reply:
I thought you specifically asked to do that in a previous PR review. Will apply proposed changes.

Reply:
Great! Hmm, not sure offhand which review that was, but if you link it, we can talk through what might have been different in that case (or if I provided bad advice).